webapp/pages/post/_id/_slug/index.vue

```diff
@@ -206,49 +215,70 @@ export default {
   },
 }
 </script>
-
 <style lang="scss">
-.page-name-post-id-slug {
+.post-page {
+  &.--blur-image > .ds-card-image img {
+    filter: blur(22px);
+  }
+
+  .ds-card-content {
+    position: relative;
+    padding-top: 24px;
+  }
+
+  .blur-toggle {
+    position: absolute;
+    top: -80px;
+    right: 0;
+
+    display: flex;
+    align-items: center;
+
+    height: 80px;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:27:04Z" title="Tuesday, December 3rd 2019, 6:27:04 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -150,6 +177,18 @@ export default {
     },
   },
   methods: {
+    unBlur() {
+      if (this.post.checkedBlur) {
+        this.post.checkedBlur = false
+        this.blur = true
+      }
+    },
+    Blur() {
+      if (!this.post.checkedBlur) {
+        this.post.checkedBlur = true
+        this.blur = false
```


```suggestion
        this.blurred = false
```